packages/metablocks/package.json

```diff
@@ -1,12 +1,36 @@
 {
   "name": "@kyraa/metablocks",
   "version": "0.0.3",
-  "main": "./dist/index.js",
   "license": "MIT",
+  "main": "dist/index.js",
+  "typings": "dist/index.d.ts",
```

Comment:
Also add github repo url here so the readme starts reflection on NPM.

Reply:
done